### ISSUE TITLE
Unit test test-stats-htest is failing with object 'extra' not found.

### ISSUE THREAD

**Issue by bruc** [tidymodels/broom#670]
R CMD check is failing with the broom package. I'm using R-3.5.2 and broom 0.5.1. The error is also being reported in the automated package testing:

[ftp://ftp.gnome.org/mirror/CRAN/web/checks/check_results_broom.html](url)

 The error is as follows:
```
* checking tests ... ERROR
  Running ‘test-all.R’
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  object 'extra' not found
  1: oneway.test(extra ~ group, data = sleep) at testthat/test-stats-htest.R:14
  2: eval(m, parent.frame())
  3: eval(m, parent.frame())
  4: stats::model.frame(formula = extra ~ group, data = sleep)
  5: model.frame.default(formula = extra ~ group, data = sleep)
  6: eval(predvars, data, env)
  7: eval(predvars, data, env)
  
  ══ testthat results  ═══════════════════════════════════════════════════════════
  OK: 729 SKIPPED: 17 FAILED: 1
  1. Error: tidy.htest/oneway.test (@test-stats-htest.R#14) 
  
  Error: testthat unit tests failed
  Execution halted
* DONE
Status: 1 ERROR, 2 NOTEs
```

**Comment by IndrajeetPatil** [comment]
Yes, this is being addressed in a new branch (https://github.com/tidymodels/broom/tree/0.5.2).

**Comment by bruc** [comment]
I have additional information for you that might help. I've been building an R environment for a customer, and using R-3.5.2 and all the packages installed as of March 11, 1030pm, the 'broom' test above worked correctly. I rebuilt the environment a few days ago, and now I get the error above. From the changed packages, I can't identify which ones might be responsible, although I've tried restoring some of the older versions of this list. I'm going to try rebuilding the environment again from the packages from March 11, and see if broom and everything else passes.
